### ISSUE TITLE
Start page

### ISSUE THREAD

**Issue by RandInaim** [fack2/Hebron-must-see#41]
we are planing to add a start page which includes pictures for each feature in our website so it would be a landing page and also redirects to the other pages

**Comment by FarahZaqout** [comment]
Good call, but I suggest you prioritize the pages of the website for now over the landing page.
